waagent

```diff
@@ -5720,6 +5767,10 @@ class Agent(Util):
             if goalState.ExtensionsConfig != None :
                 goalState.ExtensionsConfig.ReportHandlerStatus()
             
+            global LinuxDistro
+            if LinuxDistro == "redhat":
+                DoInstallRHUIRPM()
```

Comment:
[andy] Yes, but actully it will ony run once, see function DoInstallRHUIRPM(): 
 + rhuiRPMinstalled = os.path.exists(LibDir + "/rhuirpminstalled")

2701  + if rhuiRPMinstalled:

2702  + return

2703  + else:

2704  + SetFileContents(LibDir + "/rhuirpminstalled", "")